charts/rancher-vsphere-csi/rancher-vsphere-csi/2.3.0/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 annotations:
-  catalog.cattle.io/rancher-version: <2.5.99-0
+  catalog.cattle.io/rancher-version: '>= 2.5.0-0 <= 2.5.99-0'
```

Comment:
kubeVersion here should be updated to 1.20+? Or should 1.19 have both options?

Reply:
It should be okay to have both options since they are both supported (according to the official support matrix upstream) and it gives the user the option to choose CPI 1.0.100 with CSI 2.1.1 or CSI 2.3.0. CSI versions also have a different support matrix for the vSphere version so having more options is better.